### PR TITLE
Support search input type in crawler

### LINKED ISSUE #5: Support 'search' input type in the crawler
https://github.com/marco-c/autowebcompat/blob/e3f77c7b48e017a0ddfe0116e8748964d36fa74c/collect.py#L104

### PR DESCRIPTION
**Pull request by ManasviKundalia**
Fixes #5

### CODE REVIEW

**Comment by ManasviKundalia** [comment]
@marco-c On testing this code with [MDN Web docs](https://developer.mozilla.org/en-US/docs/Web)
I am getting selenium.common.exceptions.ElementNotInteractableException on line 117 of the code
`if elem.tag_name in ['button', 'a']:
elem.click()`
Please check

**Comment by marco-c** [comment]
Can you try with another test page? You could even write a simple test page yourself.

**Comment by ManasviKundalia** [comment]
@marco-c I tried with a number of other urls, that issue arises only with the MDN docs so far. Could you review my code?
Also, do we need support for input type submit? I could add that too.

File: collect.py [hunk 1]
```diff
@@ -128,6 +129,10 @@ def do_something(driver, elem_id=None):
             elem.click()
         elif input_type == 'number':
             elem.send_keys('3')
+        elif input_type == 'search':
+            elem.clear()
+            elem.send_keys('quick search')
+            elem.send_keys(Keys.RETURN)
```
Comment (marco-c) [inline collect.py]: I'm not sure about pressing return here, let's only write the text for now.

**Comment by marco-c** [comment]
> @marco-c I tried with a number of other urls, that issue arises only with the MDN docs so far. Could you review my code?

Can you upload example screenshots here that you have produced with the crawler?

> Also, do we need support for input type submit? I could add that too.

Can you file an issue about doing it? It should be a good first issue, so reserved for someone with no experience with the project.

File: collect.py (hunk 1 above)
Comment (ManasviKundalia) [reply collect.py]: Okay.

**Comment by ManasviKundalia** [comment]
@marco-c I am sorry I tried using the Resolve Conflicts option that I was getting on the file, I am not sure if it resolved the conflicts though.

**Comment by marco-c** [comment]
> @marco-c I am sorry I tried using the Resolve Conflicts option that I was getting on the file, I am not sure if it resolved the conflicts though.

Yes it did! You can also try doing it locally next time, it's easier in my opinion.

**Pull request merged**